### ISSUE TITLE
not really important, but gamesandthings is broken on steam deck

### ISSUE THREAD

**Issue by minihoot** [gamesandthings/gamesandthings.github.io#1]
![image](https://github.com/gamesandthings/gamesandthings.github.io/assets/70825884/36f22e90-9768-4eca-8ff1-2566567b6f85)
i know steam deck can run these games natively (probably) but still worth pointing out

**Comment by letsgoawaydev** [comment]
i dont own a steam deck, so unfortunately cannot test. try using chrome browser on desktop mode maybe it will work but using the native one it probably wont work

**Comment by minihoot** [comment]
yeah, also big picture mode basically emulates the steam deck ui

**Comment by letsgoawaydev** [comment]
yeah but im not sure if that also emulates the browser. will test

**Comment by letsgoawaydev** [comment]
on steam big picture the context menu doesnt work, is that what you mean by it being broken?

**Comment by minihoot** [comment]
yeah kinda
